tests/src/test/java/alluxio/client/cli/fsadmin/command/UpdateConfIntegrationTest.java

```diff
@@ -37,7 +39,7 @@ public void updateUnknownKey() {
     ret = mFsAdminShell.run("updateConf", "unknown-key");
     Assert.assertEquals(-1, ret);
     ret = mFsAdminShell.run("updateConf", "unknown-key=1=2");
-    Assert.assertEquals(-3, ret);
+    Assert.assertEquals(0, ret);
```

Comment:
Actually, see L37 returns -2 and L39 returns -1, it's weird that L41 can return 0, because you are just setting an unknown key to a different value and how will that make it succeed this time right? I figure `updateConf unknown-key=1=2` should also return non zero. And you can actually add another case where you update a known key to `key=xxx=xxx` and verify it returns 0. WDYT?